### PR TITLE
feat(components/toast): improve toast service `openComponent` `component` param type

### PR DESCRIPTION
**Pull request by Blackbaud-ErikaMcVey**

### CODE REVIEW

**Comment by blackbaud-sky-build-user** [comment]
[Storybook preview](https://blackbaud.github.io/skyux-pr-preview/667/storybook/)

Component Storybooks:

- _(no component storybooks affected in this pr)_ 

Apps:

- [code-examples](https://blackbaud.github.io/skyux-pr-preview/667/code-examples/)
- [dep-graph](https://blackbaud.github.io/skyux-pr-preview/667/dep-graph/)
- [integration](https://blackbaud.github.io/skyux-pr-preview/667/integration/)
- [playground](https://blackbaud.github.io/skyux-pr-preview/667/playground/)

**Comment by Blackbaud-ErikaMcVey** [comment]
[AB#2315251](https://dev.azure.com/blackbaud/f565481a-7bc9-4083-95d5-4f953da6d499/_workitems/edit/2315251)

File: libs/components/toast/src/lib/modules/toast/toast.service.ts [hunk 1]
```diff
@@ -71,7 +77,7 @@ export class SkyToastService implements OnDestroy {
   public openComponent(
     // TODO: change this to Type<unknown> in a breaking change to match the first
```
Comment (Blackbaud-SteveBrush) [inline libs/components/toast/src/lib/modules/toast/toast.service.ts]: Remove the TODO?

**Review by Blackbaud-SteveBrush — COMMENTED** [review]
Can we make the PR title a bit more descriptive?

**Review by Blackbaud-SteveBrush — APPROVED** [review]

**Pull request merged**